public/app/themes/justice/inc/mail-templates.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * G O V . U K   N O T I F Y
+ * -.-.-.-.-.-.-.-.-.-.-.-.-
+ * Template configuration for WordPress system
+ * Templates can be found on the GovUK Notify website:
+ * https://www.notifications.service.gov.uk/
+ */
+return [
+
+    /**
+     * Email configuration
+     ************************/
+
+    'email' => [
+        'default' => [
+            'id' => '0e73e29f-2efa-4313-a6e5-8908af51a6cc',
```

Comment:
I haven't worked with notifications before. Will this ID be consistent across all environments? If it's not, we should consdiser using an environment variable.

Reply:
Hey 😄 good question. The email templates are persistent across environments. We're good to list templates like this.